server/tserver/src/main/java/org/apache/accumulo/tserver/scan/LookupTask.java

```diff
@@ -114,8 +116,10 @@ public void run() {
           if (isCancelled())
             interruptFlag.set(true);
 
-          lookupResult = tablet.lookup(entry.getValue(), results, session.scanParams,
+          List<KVEntry> tabletResults = new ArrayList<>();
```

Comment:
I wonder if a comment here on why we are doing it this way since the reason is non-obious.